### PR TITLE
Chapter 6

### PR DESCRIPTION
**Pull request by leducson**
selection_008

### CODE REVIEW

**Comment by leducson** [comment]
ready

**Comment by leducson** [comment]
selection_017

**Comment by leducson** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  before_save self.email = email.downcase
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  validates :name, presence: true, length:
+  {maximum: Settings.validates.user.name_length}
```
Comment (hhhhuy112) [inline app/models/user.rb]: indent for this line. Check indent for this pull

**Comment by hhhhuy112** [comment]
commented

**Comment by leducson** [comment]
ready

**Comment by hhhhuy112** [comment]
commented
please rebase 2 commit

**Comment by leducson** [comment]
ready

**Comment by hhhhuy112** [comment]
merged
You can merge it

**Pull request merged**